src/ORM/Rule/ExistsIn.php

```diff
@@ -40,14 +40,24 @@ class ExistsIn
     protected $_repository;
 
     /**
+     * Construct Options
+     *
+     * @var array
+     */
+    protected $_constructOptions = [];
+
+    /**
      * Constructor.
      *
      * @param string|array $fields The field or fields to check existence as primary key.
      * @param object|string $repository The repository where the field will be looked for,
      * or the association name for the repository.
      */
-    public function __construct($fields, $repository)
+    public function __construct($fields, $repository, array $constructOptions = Array())
     {
+        $constructOptions += ['allowSqlNulls' => false];
+        $this->_constructOptions = $constructOptions;
```

Comment:
Changing. Reason was that invoke and construct can get options. This would be the difference.